frontend/public/index.html

```diff
@@ -8,6 +8,7 @@
   <link rel='stylesheet' href='build/themes.css'>
 
   <link rel="stylesheet" href="build/global.css">
+  <style></style>
```

Comment:
Why?

Reply:
The custom css will be dumped into this element

Reply:
Gotcha

Reply:
actually might be better to create a new element dunamically, you're right